strawberry/dataloader.py

```diff
@@ -58,6 +98,8 @@ def __init__(
         max_batch_size: Optional[int] = None,
         cache: bool = True,
         loop: AbstractEventLoop = None,
+        cache_map: Optional[AbstractCache] = None,
+        cache_key_fn: Optional[Callable[[K], Any]] = None,
```

Comment:
shouldn't we use T instead of any here?

Reply:
I agree on this, will make it more predictable.